debian/patches/debian.patch

```diff
@@ -0,0 +1,213 @@
+Description: Removed .sh suffixes and set NAPI_COMMON_PATH variable
+Author: Mikhail Morfikov <mmorfikov@gmail.com>
+
+--- napi-1.3.4.orig/napi.sh
++++ napi-1.3.4/napi.sh
+@@ -32,12 +32,12 @@
+ ########################################################################
+ 
+ # verify presence of the napi_common library
+-declare -r NAPI_COMMON_PATH=
++declare -r NAPI_COMMON_PATH=/usr/share/napi
+ if [ -z "$NAPI_COMMON_PATH" ] || [ ! -e "${NAPI_COMMON_PATH}/napi_common.sh" ]; then
+     echo
+-	echo "napi.sh i subotage.sh nie zostaly poprawnie zainstalowane"
++	echo "napi i subotage nie zostaly poprawnie zainstalowane"
+ 	echo "uzyj skryptu install.sh (install.sh --help - pomoc)"
+-	echo "aby zainstalowac napi.sh w wybranym katalogu"
++	echo "aby zainstalowac napi w wybranym katalogu"
+     echo
+ 	exit -1
+ fi
+@@ -191,7 +191,7 @@ declare -a g_tools=( 'tr=1' 'printf=1' '
+     'wc=1' 'dd=1' 'grep=1' 'seq=1' 'sed=1' \
+     'cut=1' 'base64=0' 'unlink=0' 'stat=1' \
+     'basename=1' 'dirname=1' 'cat=1' 'cp=1' \
+-    'mv=1' 'awk=0' 'file=0' 'subotage.sh=0' \
```

Comment:
One są zachowane w oryginalnych źródłach, to jest tylko patch dla debiana, bo tutaj pliki nie mogą mieć końcówek .sh